src/docs/faq.md

```diff
@@ -1,88 +1,101 @@
 # FAQ
 
 ::: tip
-Please read this page before asking a question in the Discord. Your help request will be ignored if the answer is found on the FAQ.
+Please read this page before asking a question on the Forum. Your help request will be pointed to the docs otherwise.
```

Comment:
why is "Forum" capitalized, also would be cool if you linked to it as well.

Reply:
Tbh if u ask anywhere we might point you to the docs, not only the forums

Reply:
I tried to leave discord out of this because we've decided to no longer do help through discord. Should I change the wording?